config.json

```diff
@@ -11,6 +11,7 @@
     "rna-transcription",
     "raindrops",
     "difference-of-squares",
+    "isogram",
```

Comment:
This problem is quite similar to panagram, but simpler, so this one should be before it.

I guess it's fine. :smile: